nautobot/extras/models/customfields.py

```diff
@@ -171,13 +206,15 @@ def get_computed_field(self, slug, render=True):
             return computed_field.render(context={"obj": self})
         return computed_field.template
 
-    def get_computed_fields(self, label_as_key=False):
+    def get_computed_fields(self, label_as_key=False, advanced_ui=None):
         """
         Return a dictionary of all computed fields and their rendered values for this model.
         Keys are the `slug` value of each field. If label_as_key is True, `label` values of each field are used as keys.
         """
         computed_fields_dict = {}
         computed_fields = ComputedField.objects.get_for_model(self)
+        if advanced_ui is not None and advanced_ui in [True, False]:
```

Comment:
I might remove the `and advanced_ui in [True, False]` check - if this function is called with a non-None, non-boolean value for `advanced_ui`, that should be an error, rather than the argument being silently ignored.